src/modules/0.0.15/aws_elasticache/entity/cache_cluster.ts

```diff
@@ -1,43 +1,42 @@
 import {
-    Column,
-    Entity,
-    JoinTable,
-    ManyToMany,
-    PrimaryColumn,
-    PrimaryGeneratedColumn,
-  } from 'typeorm'
-  
-  
-  import { cloudId, } from '../../../../services/cloud-id'
-  import { SecurityGroup, } from '../../aws_security_group/entity';
+  Column,
+  Entity,
+  JoinTable,
+  ManyToMany,
+  PrimaryColumn,
+  PrimaryGeneratedColumn,
+} from "typeorm";
 
-  export enum Engine {
-    MEMCACHED = "memcached",
-    REDIS = "redis",
-  }  
-  
-  @Entity()
-  export class CacheCluster {
-    @PrimaryColumn({
-      nullable: false,
-      type: 'varchar',
-    })
-    @cloudId
-    clusterId: string;
-      
-    // TODO: convert it to an independent table in the future
-    @Column()
-    nodeType: string;
-    
-    @Column({
-      type: 'enum',
-      enum: Engine,
-      default: Engine.REDIS
-    })
-    engine: Engine;
+import { cloudId } from "../../../../services/cloud-id";
+import { SecurityGroup } from "../../aws_security_group/entity";
 
-    @Column({
-      nullable: true,
-    })
-    numNodes?: number;
-  }
\ No newline at end of file
+export enum Engine {
+  MEMCACHED = "memcached",
+  REDIS = "redis",
+}
+
+@Entity()
+export class CacheCluster {
+  @PrimaryColumn({
+    nullable: false,
+    type: "varchar",
+  })
+  @cloudId
+  clusterId: string;
+
+  // TODO: convert it to an independent table in the future
+  @Column()
+  nodeType: string;
+
+  @Column({
+    type: "enum",
+    enum: Engine,
+    default: Engine.REDIS,
+  })
+  engine: Engine;
+
+  @Column({
+    nullable: true,
+  })
+  numNodes?: number;
```

Comment:
if this one can be nullable, remember adding the fn to convert from null to undefined like this one https://github.com/iasql/iasql-engine/blob/main/src/modules/0.0.15/aws_ec2/entity/instance.ts#L99-L107